app/models/episode_story_handler.rb

```diff
@@ -0,0 +1,95 @@
+require 'episode'
+
+class EpisodeStoryHandler
+  include PRXAccess
+
+  attr_accessor :episode, :story
+
+  def initialize(episode)
+    self.episode = episode
+  end
+
+  def self.create_from_story!(story)
+    series_uri = story.links['series'].href
+    podcast = Podcast.find_by!(prx_uri: series_uri)
```

Comment:
currently we would have to set it up ahead of time, but soon I want to have a better way.